src/Appwrite/Enum/MessageStatus.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Appwrite\Enum;
+
+enum MessageStatus: string
```

Comment:
Will we have more enums for `Message` or other entities? Maybe the structure should be Appwrite\Enum\[Entity]\[Type] or `Appwrite\Enum\Message\Type`?

Reply:
I tihnk it makes mroe sense to have it under the namespace where its actually used.

We already have `Appwrite\Messaging`

Reply:
> Will we have more enums for Message or other entities? Maybe the structure should be Appwrite\Enum[Entity][Type] or Appwrite\Enum\Message\Type?

@eldadfux, yes, we can have more. For example, `providerType` can be `email`, `sms`, or `push`.

> I think it makes more sense to have it under the namespace where it actually follows the story.
> 
> We already have `Appwrite\Messaging` for that imo

@TorstenDittmann, that path is for Realtime, though 😅. It's probably most related to [`Appwrite\Utopia\Response\Model\Message`](https://github.com/appwrite/appwrite/blob/1.5.x/src/Appwrite/Utopia/Response/Model/Message.php).

Reply:
I am kinda against having a separate namespace for enums, it's always nice if stuff like that is relative to its consuming classes (if there are).

However, we don't have enough enums implemented to know how it will look like.